node/src/index.ts

```diff
@@ -261,6 +261,39 @@ export interface Table<T = number[]> {
    */
   delete: (filter: string) => Promise<void>
 
+  /**
+   * Update rows in this table.
+   *
+   * This can be used to update a single row, many rows, all rows, or
+   * sometimes no rows (if your predicate matches nothing).
+   *
+   * @param filter  A filter in the same format used by a sql WHERE clause. The
+   *                filter may be empty, in which case all rows will be updated.
```

Comment:
More specifically, `null` right?
```suggestion
   *                filter may be null, in which case all rows will be updated.
```